Add how to migrate to ODK .md and link to it

StroemPhi opened the pull request:
This PR adds the `odk-migrate-to-odk.md` to the `docs/howto` folder to provide a How To article for the migration of an existing ontology repository to ODK, based on this draft https://docs.google.com/document/d/10i76WqZ8EbnW4ta7BBNqTOYH2tLx51QDzhWF63-80uY/. 

CC:
@matentzn
@cmungall

gouttegd commented on docs/howto/odk-migrate-to-odk.md:
```diff
@@ -0,0 +1,310 @@
+## Contributors
+
+- [Philip Strömert](https://orcid.org/0000-0002-1595-3213)
+- [Nicolas Matentzoglu](https://orcid.org/0000-0002-7356-1779)
```
Should we mention the GitHub handles as well? Given that the ODK is maintained there, the GitHub handle seems to me at least as relevant as the ORCID, maybe even more.

No strong opinion though, I’m fine with just the ORCIDs as well.

StroemPhi replied:
I don't mind. I just oriented myself on some other OBOOK articles.

gouttegd reviewed (changes requested):
Overall looks good to me, but I believe there are issues with the indenting of keys in the ODK config file shown as example. This must be fixed as people are likely going to be copy-paste that example to “bootstrap” their own migration effort.

StroemPhi commented on docs/howto/odk-migrate-to-odk.md:
````diff
@@ -0,0 +1,310 @@
+          ```
+          echo "
+          # Adding ODK runner script living in ~/tools to PATH
+          export PATH=\"\$PATH:~/tools\"" >> ~/.bashrc
````
Yes, I was expecting such a comment. Should I instead drop this, or add that this assumes one uses Bash, or is there a good how to add a folder to path permanantly link I could reference for those reader, like me, who are not so well versed in working on the command line?

StroemPhi commented on docs/howto/odk-migrate-to-odk.md:
```diff
@@ -0,0 +1,310 @@
+      export_formats:
+      - owl
+      - obo
+        reasoner: ELK
```
Thank you for catching that! I had no time to closely reread yesterday, but wanted to get the PR started.

StroemPhi commented:
> Overall looks good to me, but I believe there are issues with the indenting of keys in the ODK config file shown as example. This must be fixed as people are likely going to be copy-paste that example to “bootstrap” their own migration effort.

Thank you for the review. I've addressed all of your points and fixed some typos/improved some wording ([9d8f1af](https://github.com/OBOAcademy/obook/pull/521/commits/9d8f1af5f8b0cf650ae748253b4f893406cba719)).
